Remotion/Web/Core/ExecutionEngine/UrlMapping/IUrlMappingFileLoader.cs

```diff
@@ -0,0 +1,36 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml;
+
+namespace Remotion.Web.ExecutionEngine.UrlMapping
+{
+  /// <summary>
+  /// Provides an API for loading <see cref="UrlMappingEntry"/>s from a specified URL mapping file.
+  /// </summary>
+  public interface IUrlMappingFileLoader
```

Comment:
please add the threadsafety note unelss there's a reason it's not needed threadsafe.
(and the seealso is always nice)